apps/doji-frontend/src/pages/session-v2/[id]/index.tsx

```diff
@@ -0,0 +1,178 @@
+import { httpClient } from '@frontend/services'
+import { ExtendedNextPage } from '@frontend/type'
+import { IPublicSessionReviewDTO, IReviewStat, ISessionInformationDTO } from '@libs/api'
+import {
+  Avatar,
+  Button,
+  CircularProgress,
+  Grid,
+  LinearProgress,
+  Link,
+  Stack,
+  TextField,
+  Typography,
+  useTheme,
+} from '@mui/material'
+import { Box } from '@mui/system'
+import { formatDistanceToNow } from 'date-fns'
+import { useRouter } from 'next/router'
+
+import React from 'react'
+import { AiFillStar, AiOutlineStar } from 'react-icons/ai'
+import { BsThreeDotsVertical } from 'react-icons/bs'
+import { useQuery } from 'react-query'
+
+function SessionDetail() {
+  return (
+    <Box>
+      <Stack direction="row" py="0.5em" justifyContent="space-between">
+        <Stack spacing="0.5em">
+          <Typography variant="large" fontWeight={700}>
+            From 1M to 1K, How did I do it?
+          </Typography>
+          <Stack direction="row" spacing="0.5em">
+            <Avatar src="/static/images/avatar/1.jpg" sx={{ width: 24, height: 24 }} />
+            <Typography variant="small" fontWeight={400}>
+              by{' '}
+              <Typography variant="small" fontWeight={500}>
+                Poravee Binhayeearason
+              </Typography>
+            </Typography>
+          </Stack>
+        </Stack>
+        <Stack textAlign="right">
+          <Typography variant="large" fontWeight={700} color="primary.dark">
+            250
+          </Typography>
+          <Typography color="primary.dark">/hr/person</Typography>
+        </Stack>
+      </Stack>
+      <Typography>
+        Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nisl, aliquet fringilla sed vitae
+        amet nisl. Nullam donec molestie duis eu, neque convallis ipsum, congue.
+      </Typography>
+    </Box>
+  )
+}
+
+function RatingPanel(props: { reviewStat: IReviewStat }) {
```

Comment:
It's great to create subcomponent for this place. But please follows [1 file 1 component] to avoid large code in 1 file

Reply:
Ok